components/03-organisms/site-header/_site-header.scss

```diff
@@ -66,19 +82,19 @@ $site-header-animation-speed: 800ms;
   flex-direction: column;
 
   @media (max-width: tokens.$break-mobile-max) {
-    @include tokens.animate(max-height, $site-header-animation-speed);
+    @include tokens.animate(transform, $site-header-animation-speed);
 
-    z-index: 1;
     position: absolute;
     left: 0;
     background: var(--color-background);
-    max-height: 100vh;
+    max-height: calc(100vh - var(--header-height-mobile));
```

Comment:
This is much more noticeable when the border thickness value is set to `8` but I believe this line should be `max-height: calc(100vh - var(--header-height-mobile) + var(--site-header-border-thickness));`

Reply:
I updated the css to keep the header border in sight. Hoping that's what you were trying to achieve.